.github/workflows/deploy-pages.yaml

```
@@ -0,0 +1,79 @@
+  # Deployment job
+  deploy:
+    environment:
+      name: github-pages
+      url: ${{ steps.deployment.outputs.page_url }}
+    needs: build
+    runs-on: ubuntu-latest
+    name: Deploy
+    steps:
+      - name: Deploy to GitHub Pages
+        id: deployment
+        uses: actions/deploy-pages@d6db90164ac5ed86f2b6aed7e0febac5b3c0c03e # v4.0.5
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)